pkg/driver/redis/device_manager_redis.go

```diff
@@ -658,6 +664,9 @@ func (d *DeviceManager) GetCerts(u uuid.UUID) ([]byte, error) {
 	// hold our config
 	var b []byte
 	data, err := d.client.HGet(deviceAttestCertsHash, u.String()).Result()
+	if err != nil && errors.Is(err, redis.Nil) {
+		return nil, &common.NotFoundError{Err: fmt.Sprintf("device not found %s", u)}
```

Comment:
Maybe `attestation certificates for the device %s not found` will better describe the error here?

Reply:
Sure, I like this comment and the next. Adding them now.

Reply:
Done both.